Makefile.common.mk

```diff
@@ -5,7 +5,7 @@
 # common-files repo, make the change there and check it in. Then come back to this repo and run
 # "make updatecommon".
 
-# Copyright 2018 Istio Authors
```

Comment:
Shouldn't this stay as the original date?

Reply:
Per CNCF recommendation, we'll be removing the date entirely in new copyright notices. In fact, this has been done in the common-files repo and will be replicated here the next time the common files are updated.